kvrocks.conf

```diff
@@ -97,6 +97,16 @@ dir /tmp/kvrocks
 #
 log-dir stdout
 
+# You can configure log-retention-days to control whether to enable the log cleaner
+# and the maximum retention days that the INFO level logs will be kept.
+#
+# if set to -1, that means to disable the log cleaner.
+# if set to 0, all previous INFO level logs will be immediately removed.
```

Comment:
I read we use the same config for all logs. How can logs in other levels have different manner?

Reply:
> @IoCing Would you mind adding Go test cases for this configuration, to make sure it can be changed in-flight?

ok , i will try

Reply:
> I read we use the same config for all logs. How can logs in other levels have different manners?

At present, glog does not provide an interface to configure the expiration time of different levels
I think it is possible to submit a pr to glog, and then we can set up accordingly。

Reply:
@IoCing I think we can move on with all logs with the same retention behavior first, then optimize it if necessary.